2007-07-26  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/sparc/tls.h (tcbhead_t): Add gscope_flag.

--- nptl/sysdeps/sparc/tls.h.orig	2007-07-26 00:36:08.000000000 +0200
+++ nptl/sysdeps/sparc/tls.h	2007-07-26 00:36:25.000000000 +0200
@@ -46,6 +46,7 @@
   dtv_t *dtv;
   void *self;
   int multiple_threads;
+  int gscope_flag;
   uintptr_t sysinfo;
   uintptr_t stack_guard;
   uintptr_t pointer_guard;
